qt/scientific_interfaces/Indirect/IndirectFitOutput.cpp

```diff
@@ -217,6 +217,64 @@ void renameResult(WorkspaceGroup_sptr resultWorkspace,
     renameResultWithoutSpectra(resultWorkspace, fitDataBegin, fitDataEnd);
 }
 
+std::vector<std::string> strsplit(const std::string &string_in,
```

Comment:
Boost can do this (see https://www.boost.org/doc/libs/1_71_0/doc/html/string_algo/usage.html#id-1.3.3.5.9).